packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -940,6 +940,8 @@ export const vivifyVaultManagerKit = baggage => {
             want: { Minted: null },
           });
 
+          // NB: This increments even when a vault fails to init and is removed
+          // from the manager, creating a sparse series of published vaults.
```

Comment:
ouch... ISTR from a security audit that erroneous inputs are not supposed to cause any change in state.

Well, better to have it documented, I suppose.